frontend-v2/src/features/trial/Doses.tsx

```diff
@@ -32,6 +32,7 @@ const Doses: FC<Props> = ({ project, protocol, units }) => {
       { id: protocol.variables[0] || 0 },
       { skip: !protocol.variables.length },
     );
+  const mappedVariable = protocol.mapped_qname || variable?.name || '';
```

Comment:
Should we be consistent about using either `qname` or `name` for variables here?

```suggestion
  const mappedVariable = protocol.mapped_qname || variable?.qname || '';
```